rd_ui/bower.json

```diff
@@ -33,7 +33,8 @@
     "angular-ui-select": "0.8.2",
     "angular-bootstrap-show-errors": "~2.3.0",
     "angular-sanitize": "1.2.18",
-    "d3": "3.5.6"
+    "d3": "3.5.6",
+    "ng-tags-input": "2.1.1"
```

Comment:
We're already using `angular-ui-select` which I believe has comparable features, see this example: http://plnkr.co/edit/juqoNOt1z1Gb349XabQ2?p=preview